__test__/server/texter.test/texter.test.js

```diff
@@ -156,11 +156,12 @@ it("should be able to receive a response and reply (using fakeService)", async (
       expect.objectContaining({
         send_status: "DELIVERED",
         text: `responding to ${message.text}`,
-        user_id: testTexterUser.id,
+        user_id: null,
         contact_number: testContact.cell,
         campaign_contact_id: testContact.id
       })
     );
+    expect(dbMessage[1].is_from_contact).toBeTruthy();
```

Comment:
I could add this as =true above, but sqlite annoyingly returns as `1` instead, so we need to be truthy here.